### PR TITLE
ENH: Add utility function to quickly check FS license

### LINKED ISSUE #502: FreeSurfer canary test implementation
Implements poldracklab/smriprep#43

### PR DESCRIPTION
**Pull request by mgxd**
Closes #502

### CODE REVIEW

**Comment by effigies** [comment]
Does the context manager chdir? Off the top of my head, I don't think it does.

**Comment by mgxd** [comment]
you are right, i've changed it accordingly

File: niworkflows/utils/misc.py [hunk 1]
```diff
@@ -263,5 +263,30 @@ def pass_dummy_scans(algo_dummy_scans, dummy_scans=None):
     return dummy_scans
 
 
+def check_valid_fs_license():
+    """Quickly runs mri_convert to weed out FreeSurfer license issues"""
```
Comment (oesteban) [inline niworkflows/utils/misc.py:267]:
```suggestion
    """Run ``mri_convert`` to assess FreeSurfer access to a license."""
```

**Comment by oesteban** [comment]
bumping up - what's the status here?

**Comment by mgxd** [comment]
this fell off my radar, I'll finish it up today

**Pull request merged**